cmd/serv.go

```diff
@@ -126,25 +127,35 @@ func runServ(c *cli.Context) error {
 		return nil
 	}
 
-	verb, args := parseCmd(cmd)
```

Comment:
`parseCmd` is now unused, and causing linter to fail, the function can be removed.

Reply:
thanks I missed that.